### ISSUE TITLE
[Docu] get docker ip address

### ISSUE THREAD

**Issue by Gummibeer** [LaraDock/laradock#67]
```shell
$ ifconfig docker0 | grep 'inet' | cut -d: -f2 | awk '{ print $1}' | head -n1
```

With this you will get only the IP-Address without any other informations.

**Comment by Mahmoudz** [comment]
@Gummibeer fantastic 👍 
Would you like to send a PR updating the readme, or I do it myself?

**Comment by Gummibeer** [comment]
@Mahmoudz PR is done - check if it is ok for you. ;)

**Comment by Mahmoudz** [comment]
@Gummibeer Thank u.

Closed in #68
